app/models/EmailTemplate.scala

```diff
@@ -40,7 +43,12 @@ object EmailTemplate {
   private val CUSTOMS_DUTY_AND_IMPORT_VAT_DUE_DATE = 16
   private val DD1920_STT_DUE_DATE                  = 25
   private val DD1720_STT_DUE_DATE                  = 15
-  private val emailDateFormatter                   = DateTimeFormatter.ofPattern("dd MMM yyyy")
+
+  val emailDateFormatter: DateTimeFormatter = DateTimeFormatterBuilder()
+    .appendPattern("dd ")
```

Comment:
Better to mention the space explicitly (like below) to misinterpret as unwanted space. Better to make use of singleSpace. Same at line 50 and same for Utils. httpDateFormatter

s"dd$singleSpace"